app/models/concerns/hyrax/ability/collection_ability.rb

```diff
@@ -91,9 +116,7 @@ def collection_roles
           models.each do |collection_model|
             # Permit all actions (same collection permissions as admin users)
             can :manage, collection_model
-            can :manage, ::SolrDocument do |solr_doc|
-              solr_doc.collection?
-            end
+            can :manage, ::SolrDocument, &:collection?
```

Comment:
I've never seen this syntax before, it's interesting. I think this change is good, I would just want to double check that it's accomplishing the same thing it was before (we can do this together)

Reply:
This one was an autocorrect one. Yes, let's look at this together!